kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```diff
@@ -615,6 +615,46 @@ paths:
                 items:
                   $ref: '#/components/schemas/ConsumerGroup'
 
+  /api/clusters/{clusterName}/consumer-groups/paged:
```

Comment:
Why do not use the same url with old-style/ new style behavior?

Reply:
in paging endpont we need to return total number of pages, so return types are incompatible